frontend/src/app/components/ExperimentTourDialog/index.js

```diff
@@ -6,44 +6,85 @@ import React from "react";
 
 import StepModal from "../StepModal";
 
+import type { InstalledExperiments } from "../../reducers/addon";
+
+import { getBreakpoint } from "../../containers/App";
 import { experimentL10nId } from "../../lib/utils";
 
 type ExperimentTourDialogProps = {
+  hasAddon?: boolean,
+  installed?: InstalledExperiments,
   experiment: Object,
   isExperimentEnabled: Function,
   onCancel: Function,
   onComplete: Function,
-  sendToGA: Function
+  sendToGA: Function,
+  isFeatured?: boolean
 }
 
 export default class ExperimentTourDialog extends React.Component {
   props: ExperimentTourDialogProps
 
+  applyAdditionalMetricsArgs(gaArgs: Object) {
+    const { experiment, installed, hasAddon } = this.props;
+    return Object.assign(gaArgs, {
+      dimension1: hasAddon,
+      dimension2: installed ? Object.keys(installed).length > 0 : false,
+      dimension3: installed ? Object.keys(installed).length : 0,
+      dimension10: getBreakpoint(window.innerWidth),
+      dimension11: experiment.slug,
+      dimension13: "Featured Experiment"
+    });
+  }
+
   stepNextPing = (newStep: number) => {
-    this.props.sendToGA("event", {
+    const { sendToGA, isFeatured } = this.props;
+
+    let gaArgs = {
       eventCategory: "ExperimentDetailsPage Interactions",
       eventAction: "button click",
       eventLabel: `forward to step ${newStep}`,
-      dimension11: this.props.experiment.slug
```

Comment:
Why are we deleting `dimension11`? The docs certainly make it seem like it should be included: https://github.com/mozilla/testpilot/blob/master/docs/metrics/ga.md#custom-dimensions

Reply:
You're correct, I'll add that one back in